install/kubernetes/helm/istio-remote/templates/configmap.yaml

```diff
@@ -76,12 +76,29 @@ data:
       # Set concurrency to a specific number to control the number of Proxy worker threads.
       # If set to 0 (default), then start worker thread for each CPU thread/core.
       concurrency: {{ .Values.global.proxy.concurrency }}
-
-    {{- if .Values.global.remoteZipkinAddress }}
       #
-      # Zipkin trace collector
-      zipkinAddress: {{ .Values.global.remoteZipkinAddress }}:9411
-    {{- end }}
+      tracing:
+      {{- if eq .Values.global.proxy.tracer "lightstep" }}
+        lightstep:
+          # Address of the LightStep Satellite pool
+          address: {{ .Values.global.tracer.lightstep.address }}
+          # Access Token used to communicate with the Satellite pool
+          accessToken: {{ .Values.global.tracer.lightstep.accessToken }}
+          # Whether communication with the Satellite pool should be secure
+          secure: {{ .Values.global.tracer.lightstep.secure }}
+          # Path to the file containing the cacert to use when verifying TLS
+          cacertPath: {{ .Values.global.tracer.lightstep.cacertPath }}
+      {{- else if eq .Values.global.proxy.tracer "zipkin" }}
+        {{- if .Values.global.tracer.zipkin.address }}
+        zipkin:
```

Comment:
minor: The `zipkin:` and following comment line should be factored out above the `{{- if`, as common in both paths.